organic-search-analytics.sql

```diff
@@ -66,8 +66,8 @@ CREATE TABLE IF NOT EXISTS `search_analytics` (
   `impressions` int(11) NOT NULL,
   `clicks` int(11) NOT NULL,
   `ctr` float NOT NULL,
-  `avg_position` int(11) NOT NULL,
-  `avg_position_click` int(11) NULL,
+  `avg_position` float NOT NULL,
+  `avg_position_click` float NULL,
```

Comment:
These changes look smart and safe.  However this will only fix new installs of this tool.

There is an upgrade script that this change will need to be added to which will allow existing installs of the tool to receive this change.
https://github.com/PromInc/organic-search-analytics/blob/master/organic-search-analytics/upgrade.php

An individual user would then need to run the upgrade step manually after updating their code.  That will correct the database and allow for future data to benefit from this change.  Additionally, the user could delete up to 90 days of data and re-import their data.  This can all be done via the home page.